Pipelines/ci.yaml

```diff
@@ -17,6 +17,11 @@ resources:
     endpoint: ToolsAccess
     name: tools.internal
     ref: 78ecf591964ecf511b4c7a0cd84d98ce11e8be16
+  - repository: DocToolUnityProject
+    type: git
+    endpoint: ToolsAccess
+    name: lib.doctools
+    ref: refs/heads/main
```

Comment:
Do we want to lock this into a specific commit, so we aren't potentially inadvertently broken by a change to the repo? That way, any change made in the lib.doctools repo will need to be explicitly adopted here, which feels a bit more stable.

Reply:
I thought about that, but I sort of want to use MRTK3 as the "CI" for that repo to catch inadvertent changes to that repo.

Reply:
I'd be happy to work with you to set up a real CI for that repo 😀 I'd prefer not to potentially break MRTK's pipelines based on someone making an inadvertent change to a different repo.

Reply:
Sounds great! I have updated the yaml files to point to a specific commit.